core/src/main/java/com/netease/arctic/optimizing/RewriteFilesInput.java

```diff
@@ -32,6 +32,7 @@
 import java.util.stream.Collectors;
 
 public class RewriteFilesInput extends BaseOptimizingInput {
+  private static final long serialVersionUID = -696610318564333923L;
```

Comment:
Maintaining a manual `serialVersionUID` for RewriteFilesInput is necessary as the serialized result of this class will be stored, allowing for potential cross-version usage. 

However, the change seems to bring about compatibility issues during the upgrade process. How about calculating the default `serialVersionUID` value for the class for now and declaring it to prevent upgrade issues?

Reply:
The serialization ID is generated based on the current code, and it is the same as the serialization ID generated by the JDK by default.

Reply:
Nice catch, that is what I expected.